io/genbank/genbank_test.go

```diff
@@ -802,3 +813,11 @@ func TestConsortiumRegression(t *testing.T) {
 		t.Errorf("Failed to read consrtm. Got err: %s", err)
 	}
 }
+
+// testcase for subset of unusual file
+// includes implicit genome range with partial
+// and origin is replaced with contig
+func TestParseS288C_IX(t *testing.T) {
+	_, err := ReadMulti("../../data/NC_001141.2_redux.gb")
+	assert.Nil(t, err)
+}
```

Comment:
Specifically mention that this is a regression test for that particular test case.